service/service.go

```diff
@@ -269,8 +273,10 @@ func (s *Service) Start(b Broadcaster) error {
 			open := time.Now()
 			// TODO(dshulyak) some time is wasted here to persist data on disk
 			// on my computer ~20ms
-			s.openRound = s.newRound(s.prevRound.Epoch() + 1)
-			log.Info("Round %v opened. took %v", s.openRound.ID, time.Since(open))
+			s.openRoundMutex.Lock()
+			s.newRound(s.prevRound.Epoch() + 1)
```

Comment:
Must the whole invocation of `s.newRound` be in a critical section?
If not, consider returning `*round`  and do the assignment under a lock (or use atomic)
If yes, consider documenting that fact in the method's comment.